src/NSwag.Generation.WebApi/Processors/OperationParameterProcessor.cs

```diff
@@ -182,8 +182,18 @@ public bool Process(OperationProcessorContext context)
 
                 if (operationParameter != null)
                 {
-                    operationParameter.Position = position;
                     position++;
+                    operationParameter.Position = position;
+
+                    if (_settings.SchemaType == SchemaType.OpenApi3)
+                    {
+                        operationParameter.IsNullableRaw = null;
```

Comment:
What does this mean ?

Reply:
This is a fix to align web api impl with asp.net core - not really part of this PR.